example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.21'
     repositories {
```

Comment:
I suggest to bump `build.gradle` plugin on line 9 to `7.2.1` as well, because `3.5.0` is very old at the moment.
It is not that old to be not supported by current Android Studio builds, but still, the update would be a good future-proofing for example as well.

Reply:
thx! Noted and will do that!